forwarder.go

```diff
@@ -192,13 +195,37 @@ func stripEmptyStrings(eventlist []string) []string {
 }
 
 func writeJSON(eventlist []string) string {
-	//Function produces Splunk HEC compatible json document for batched eventss
+	//Function produces Splunk HEC compatible json document for batched events
 	// Example: { "event": "event 1"} { "event": "event 2"}
-	jsonPREFIX := "{ \"event\":"
-	jsonPOSTFIX := "}"
-	jsonDOC := strings.Join(eventlist, "} { \"event\":")
-	jsonDOC = strings.Join([]string{jsonPREFIX, jsonDOC, jsonPOSTFIX}, " ")
-	return jsonDOC
+	var jsonDoc string
+
+	for _, e := range eventlist {
+		var timestamp []string
+
+		if timestampRegex == nil {
```

Comment:
If you use `MustCompile` to initialise timestampRegex in `init()`, then this can never be nil.